client/profile-wizard/index.js

```diff
@@ -337,9 +335,16 @@ export default compose(
 		} = dispatch( PLUGINS_STORE_NAME );
 		const { updateNote } = dispatch( NOTES_STORE_NAME );
 		const { updateOptions } = dispatch( OPTIONS_STORE_NAME );
-		const { updateProfileItems } = dispatch( ONBOARDING_STORE_NAME );
+		const {
+			updateProfileItems,
+			invalidateResolutionForStoreSelector,
+		} = dispatch( ONBOARDING_STORE_NAME );
 		const { createNotice } = dispatch( 'core/notices' );
 
+		const clearTaskCache = () => {
+			invalidateResolutionForStoreSelector( 'getTasksStatus' );
```

Comment:
invalidate the task status cache to ensure that we get an updated task status when redirecting.